src/validate.ts

```diff
@@ -0,0 +1,36 @@
+import { isNullOrEmpty } from './utils';
+
+export interface Options {
+    APP_FILE_PATH: string;
+    SCOPE: Scope;
+    SITE_COLLECTION_URL: string;
+    SKIP_FEATURE_DEPLOYMENT: boolean;
+    OVERWRITE: boolean;
+}
+
+export enum Scope {
+    Tenant = 'tenant',
+    SiteCollection = 'sitecollection'
+}
```

Comment:
Here we also have a spacing of 4 instead of 2

```suggestion
export interface Options {
  APP_FILE_PATH: string;
  SCOPE: Scope;
  SITE_COLLECTION_URL: string;
  SKIP_FEATURE_DEPLOYMENT: boolean;
  OVERWRITE: boolean;
}

export enum Scope {
  Tenant = 'tenant',
  SiteCollection = 'sitecollection'
}
```